src/explorer/SiteTreeItem.ts

```diff
@@ -170,6 +175,29 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
     }
+
+    private async promptToSaveDeployConfigs(node: IAzureNode<SiteTreeItem>, fsPath: string): Promise<void> {
+        const saveDeploymentConfig: string = `Always deploy the workspace "${fsPath}" to "${node.treeItem.client.fullName}"?`;
+        const neverShowAgain: MessageItem = { title: "Never" };
+        let result: MessageItem = DialogResponses.learnMore;
+        while (result === DialogResponses.learnMore) {
+            result = await ext.ui.showWarningMessage(saveDeploymentConfig, DialogResponses.yes, neverShowAgain, DialogResponses.skipForNow, DialogResponses.learnMore);
+            if (result === DialogResponses.yes) {
+                workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.deploymentConfigurations, {
+                    app: node.id,
+                    path: fsPath
+                });
+            } else if (result === neverShowAgain) {
+                workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.deploymentConfigurations, {
+                    dontShowAgain: true
+                });
+            } else if (result === DialogResponses.learnMore) {
+                // tslint:disable-next-line:no-unsafe-any
+                opn('https://aka.ms/AA1pq88');
```

Comment:
I'm not sure a 'Learn more' link is needed for this. You're just saving some defaults - that seems pretty obvious to me

Reply:
I was thinking a new user wouldn't be sure what it means to save deployment configurations.  If they hit yes and couldn't deploy to anything but their first app, it could be kind of frustrating when you don't have a clear understanding as to what workspace settings are.

Reply:
I think this is a pretty complex dialog already, though, and I'm not sure it helps them when they're the most confused (aka the _next_ time they go to deploy). 

We already have a confirmation dialog for deploying, correct? What if that dialog has a 'Change defaults' button added if we detect they're using this deploy config. Or even simpler - you could just print the 'learn more' link in the deploy logs